controller/schema.mustache

```diff
@@ -1,5 +1,7 @@
 {
   "type": "object",
+  "name": "{{name}}",
```

Comment:
are we cool with adding non standard properties to json schemas? We're heading off spec with `name` and `pluralName`. Are they necessary in schema?